Gems/Atom/RPI/Code/Source/RPI.Edit/Material/MaterialTypeSourceData.cpp

```diff
@@ -46,29 +48,17 @@ namespace AZ
             if (JsonRegistrationContext* jsonContext = azrtti_cast<JsonRegistrationContext*>(context))
             {
                 jsonContext->Serializer<JsonMaterialPropertySerializer>()->HandlesType<MaterialTypeSourceData::PropertyDefinition>();
+                jsonContext->Serializer<JsonMaterialPropertyConnectionSerializer>()->HandlesType<MaterialTypeSourceData::PropertyConnection>();
+                jsonContext->Serializer<JsonMaterialPropertyGroupSerializer>()->HandlesType<MaterialTypeSourceData::GroupDefinition>();
             }
             else if (auto* serializeContext = azrtti_cast<SerializeContext*>(context))
             {
-                serializeContext->Class<PropertyConnection>()
-                    ->Version(1)
-                    ->Field("type", &PropertyConnection::m_type)
-                    ->Field("id", &PropertyConnection::m_nameId)
-                    ->Field("shaderIndex", &PropertyConnection::m_shaderIndex)
-                    ;
+                serializeContext->Class<PropertyConnection>()->Version(3);
+                serializeContext->Class<GroupDefinition>()->Version(4);
```

Comment:
Why did we jump to version 3/4 from 1? Just wondering, its not a big deal.

Reply:
I don't remember :)